sensor_msgs/Image im
int64 cameraFrameId
int8 singleBoxId
int64 sortAlgoId

float32 kindeOfStrawberry
float32 w
float32 h
float32 x
float32 y
float32 conf
